graphprot/Metrics.py

```diff
@@ -54,20 +54,23 @@ def __init__(self, y_pred, y_hat, target, threshold=4, binary=True):
 
         if self.binary == True:
 
-            self.y_pred = get_boolean(
+            y_pred_CM = get_boolean(
```

Comment:
overall the naming of the variables is a tiny bit confusing for me. `y_pred` is fine but already `y_hat` is a strange name for the ground truth. The same for the CM that I don't fully understand what it stands for. The same apply for the names of the metrics FP, FN TP and TN that if you know what they are it probably makes perfect sense but otherwise not so much, Being a bit more verbose in the naming convention here would be better I think

Reply:
I updated all the naming to make it more clear. Can you have a look ?